.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: CI-WORKFLOW
+
+on:
+  pull_request:
+    branches:
+      - '*'
+
+permissions:
+  contents: read
+
+env:
+  mock_version_python3: "5.0.1"
+  requests_version_python3: "2.28.2"
+  mock_version_python2: "3.0.5"
+  requests_version_python2: "2.27.1"
+  ACCOUNT_ID: ${{ secrets.ACCOUNT_ID }}
+  API_KEY: ${{ secrets.API_KEY }}
+
+jobs:
+  build-and-test-python3:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.10.14"
+      - name: Install test dependencies
+        run: |
+          pip install mock==5.0.1
```

Comment:
Do you want to use env `mock_version_python3` since you have that defined above?